playwright.config.ts

```diff
@@ -0,0 +1,79 @@
+import { defineConfig, devices } from '@playwright/test'
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
```

Comment:
Why do we have commented code here?

Reply:
haha autogenerated 😅 
Lemme remove it as well